diff --git a/internal/pkg/image/unpacker/squashfs.go b/internal/pkg/image/unpacker/squashfs.go
index e86ae3bb8..a58d7d8fc 100644
--- a/internal/pkg/image/unpacker/squashfs.go
+++ b/internal/pkg/image/unpacker/squashfs.go
@@ -62,7 +62,10 @@ type Squashfs struct {
 // NewSquashfs initializes and returns a Squahfs unpacker instance
 func NewSquashfs() *Squashfs {
 	s := &Squashfs{}
+	defaultPath := os.Getenv("PATH")
+	os.Setenv("PATH", os.Getenv("USER_PATH"))
 	s.UnsquashfsPath, _ = bin.FindBin("unsquashfs")
+	os.Setenv("PATH", defaultPath)
 	return s
 }
